package.json

```diff
@@ -14,10 +14,14 @@
     "gulp-uglify": "^2.0.0",
     "merge-stream": "^1.0.0",
     "plylog": "^0.4.0",
-    "polymer-build": "^0.4.1"
+    "polymer-build": "^0.4.1",
+    "polyserve": "polymerlabs/polyserve#master"
```

Comment:
Why do we need the `master` version of polyserve? Does a stable version not suffice?